cellpack/autopack/__init__.py

```diff
@@ -281,7 +281,8 @@ def is_remote_path(file_path):
     @param file_path: str
     """
     for ele in DATABASE_NAME:
-        return ele in file_path
+        if ele in file_path:
+            return ele in file_path

```

Comment:
Could just return `True` instead of making another comparison
```suggestion
            return True
```